rsts/concepts/tasks.rst

```diff
@@ -24,49 +24,46 @@ In general, a Flyte task is characterized by:
 Can "X" Be a Flyte Task?
 -------------------------
 
-When deciding whether or not a unit of execution constitutes a Flyte task, consider the following:
+When deciding if a unit of execution constitutes a Flyte task, consider these questions:
 
-- Is there a well-defined graceful/successful exit criteria for the task? A task is expected to exit after finishing processing
-  its inputs.
-- Is it repeatable? Under certain circumstances, a task might be retried, rerun, etc. with the same inputs. It's expected
-  to produce the same outputs every single time. For example, avoid using random number generators with current clock as seed
-  and instead use a system-provided clock as the seed. 
-- Is it a pure function, i.e., does it have side effects that are not known to the system (e.g. calls a web-service)? It's strongly
-  advisable to avoid side-effects in tasks. When side-effects are required, ensure that those operations are idempotent.
+- Is there a well-defined graceful/successful exit criteria for the task? A task is expected to exit after completion of input processing.
+- Is it repeatable? Under certain circumstances, a task might be retried, rerun, etc. with the same inputs. It is expected
+  to produce the same output every single time. For example, avoid using random number generators with current clock as seed. Use a system-provided clock as the seed instead. 
+- Is it a pure function, i.e., does it have side effects that are unknown to the system (calls a web-service)? It is recommended to avoid side-effects in tasks. When side-effects are evident, ensure that the operations are idempotent.
```

Comment:
Is this rendering properly?

Reply:
Yes..https://flyte--2409.org.readthedocs.build/en/2409/concepts/tasks.html#divedeep-tasks